2 - ListsLogicAndIterations/Code/Lists.cs

```diff
@@ -7,27 +7,44 @@ public static class Lists
 	{
 		public static List<decimal> BuildList(decimal first, decimal second)
 		{
-			throw new NotImplementedException();
+            List<decimal> result = new List<decimal>() { first, second };
+            return result;
+			//throw new NotImplementedException();
 		}
 
-		public static decimal GetValue(List<decimal> list, int position)
-		{
-			throw new NotImplementedException();
+        public static decimal GetValue(List<decimal> list, int position)
+        {
+            decimal result;
+            try
+            {
+                result = list[position];
+            }
+            catch (ArgumentOutOfRangeException)
+            {
+                result = 0;
+            }
+            return result;
```

Comment:
Lists have methods and properties to determine if it exists like arrays or dictionaries (though each has slightly different names and use cases). Try to avoid exceptions.